tools/dynamic/refresh.py

```
@@ -0,0 +1,142 @@
+    return {
+        "ok": True,
+        "path": rel(path),
+        "changed": changed,
+        "dry_run": dry_run,
+        "errors": [],
+        "warnings": sync_warnings,
+        "blocks": block_results,
+    }
+
+
+def sync_dynamic_block_config(path: Path, frontmatter: str, body: str) -> tuple[str, list[str]]:
```

Ruff (0.15.15)

[warning] Unused function argument: `path`

(ARG001)